src/main/java/teammates/ui/pagedata/InstructorStudentRecordsAjaxPageData.java

```diff
@@ -13,18 +14,24 @@
     private List<FeedbackResultsTable> resultsTables;
 
     public InstructorStudentRecordsAjaxPageData(AccountAttributes account, StudentAttributes student, String sessionToken,
-                                                List<FeedbackSessionResultsBundle> results) {
+                                                List<FeedbackSessionResultsBundle> results,
+                                                Map<String, Boolean> sessionSubmissionStatusMap) {
         super(account, student, sessionToken);
         this.resultsTables = new ArrayList<>();
         for (int i = 0; i < results.size(); i++) {
             FeedbackSessionResultsBundle result = results.get(i);
+            boolean submissionStatus = true;
```

Comment:
What does `submissionStatus = true` mean?

Reply:
we need to display PC explicitly if the student himself has not submitted the session, thats why I assume by default that student has submitted the session

Reply:
`submissionStatus` as a variable name does not imply that it is a boolean. Should name it something like `hasStudentSubmitted`.